docs/source/faq.rst

```diff
@@ -21,6 +21,16 @@ Is gabbi only for testing Python-based APIs?
 No, you can use :doc:`gabbi-run <runner>` to test an HTTP service
 built in any programming language.
 
+Workarounds
+~~~~~~~~~~~
+
+pytest produces warnings about yield tests. Can I make them stop?
```

Comment:
Oh never mind, I see, you're saying exactly what I'm saying, I just haven't adapted to where you put the comment. I would have put it the line below.

Anyway, fix it please, or at least make a bug, this has already merged.